### PR TITLE
[WIP] Added GATT CGM Specific Ops Control Characteristic

### PR DESCRIPTION
**Pull request by carlos21**
**Issue**

#58 

**What does this PR Do?**

Added GATT CGM Specific Ops Control Characteristic

**Where should the reviewer start?**

GATTCGMSpecificOpsControl.swift

**Sweet giphy showing how you feel about this PR**

Giphy

### CODE REVIEW

**Comment by carlos21** [comment]
@colemancda I'm not sure how to implement the E2ECRC field 'cause it's not mandatory and is in the middle of the rest xD

File: Sources/Integer.swift [hunk 1]
```diff
@@ -27,7 +27,7 @@ internal extension Int {
     }
 }
 
-internal extension UInt16 {
+public extension UInt16 {
```
Comment (colemancda) [inline Sources/Integer.swift]: This needs to stay `internal` to avoid collision with other modules.

**Comment by colemancda** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/6e0cf35b82c539dc05c1f7e5a7cc33bc/6e0cf35b82c539dc05c1f7e5a7cc4179)
---
*Duration*: 12 minutes and 24 seconds
*Result*: **Perfect build!** All 183 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/6e0cf35b82c539dc05c1f7e5a7cc33bc/6e0cf35b82c539dc05c1f7e5a7cc4179)
---
*Duration*: 12 minutes and 24 seconds
*Result*: **Perfect build!** All 183 tests passed. :+1:
*Test Coverage*: 86%

**Pull request closed without merging**